arch/arm/mach-imx/imx_secondary_boot.c

```diff
@@ -100,3 +102,4 @@ U_BOOT_CMD(
 	"Assert WDOG Software Reset Signal (internal reset) via TF-A",
 	"\n"
 );
+
```

Comment:
Cosmetic, no need for the empty line here.

Reply:
sure, will fix